reflectapi-demo/clients/rust/Cargo.toml

```diff
@@ -6,9 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-reflectapi-demo-client-generated = { path = "generated", features = [
-    "reqwest",
-] }
+reflectapi-demo-client-generated = { path = "generated" }
```

Comment:
the content of the nested Cargo file could be here directly 
![image](https://github.com/user-attachments/assets/cd4ef99e-156f-4698-beff-27b54b5eba19)

Reply:
Do you mean flattening `reflectapi-demo-client-generated` into ``reflectapi-demo-client`?